components/gitpod-db/src/typeorm/migration/1675433412120-UserOrganizationId.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists, indexExists } from "./helper/helper";
+
+const table = "d_b_user";
+const column = "organizationId";
+const orgIndex = "idx_organizationId";
+
+export class UserOrganizationId1675433412120 implements MigrationInterface {
```

Comment:
In this PR, there's a mix of schema changes, and applications using it. In a meeting on Tue Jan 31st, we agreed we wouldn't lump schema and application changes together as it prevents us from ensuring that schema changes are always applied before application changes which use those - crucial for Dedicated.

Is it possible to split this into just the schema update, which is backwards compatible, and then the application changes on top?